src/call/pairwise.rs

```diff
@@ -189,7 +189,15 @@ pub fn call<A, B, P, M, R, W, X, F>(
             for (i, event) in events.iter().enumerate() {
                 for (j, pileup) in pileups.iter().enumerate() {
                     let p = if let &Some(ref pileup) = pileup {
-                        Some(pileup.posterior_prob(&event.af_case, &event.af_control))
+                        // if either of the pileups is empty, no valid probability could be
+                        // calculated for it -- and in SingleCellBulkModel's joint_prob() and map()
+                        // this would result in a division by zero and thus a program crash
+                        if  !pileup.case_observations().is_empty() &&
+                            !pileup.control_observations().is_empty() {
+                                Some(pileup.posterior_prob(&event.af_case, &event.af_control))
+                            } else {
+                                None
+                            }
```

Comment:
Hi, sorry for the delay, we had two deadlines yesterday. I would have time today. If you like, please choose a time before 3pm.